### ISSUE TITLE
v1.44 - 3080Ti - GPU is lost. Reboot the system to recover this GPU

### ISSUE THREAD

**Issue by charnet3d** [Lolliedieb/lolMiner-releases#1197]
Hi,
Today I have been plagued by this issue. While testing some higher frequencies around 1500mhz (+/- 60mhz) to find the perfect balance of ETH and TON, I started losing the gpu. My setup is just a gaming PC with a 1070 on 2nd slot that I mine on and also use for the display when mining (to improve LHR stability).

Frankly I don't remember if I had the gpu disconnection starting to occur right before I upgraded to 1.44 or after. But one thing is sure: The last 3 hours I've been trying to make v1.44 work but I keep encountering this issue after some time (or very quickly at high frequencies like 1600mhz).

I reverted now to 1.43 and it's been mining away very stable for 30mins+ at 1500mhz, giving 83.85Mh/s and 2.584Gh/s.

From the Nvidia forums I saw that gpus disappearing like that could be either bad thermals or the psu. As my card runs quite cool, core never exceeding 60c when mining and I had the pads changed for the mem, which makes it run at < 90c at high fan RPM. This leaves the power supply issue. But as now 1.43 is working fine, I'm thinking maybe there is some kind of spiky power usage happening on the newer version that makes the gpu fail hard and disconnect itself.

We need more people to shime in to be able to root out the cause of this.
Also the youtuber CryptoDonkeyMiner had a similar thing happen, https://www.youtube.com/post/UgkxBsVSd7eK57eMFm3NTq2VaR9Sc5iJ7xiH
He said he was testing ETH+ALPH where he noticed only one card showing, but after rebooting hiveOS he couldn't post, he had to check his rig and then he concluded that his 3080Ti is dead (similar to my issue, even bios couldn't recognize it anymore and the leds not turning on. I could only resurrect my card when I disconnect all power for a few minutes) CryptoDonkey had to test his card on another machine to finally see that it wasn't dead.
This COULD be caused by lolminer if it was run before on that card or maybe the same bug on Trex.

**Comment by jgonzis** [comment]
> Hi, Today I have been plagued by this issue. While testing some higher frequencies around 1500mhz (+/- 60mhz) to find the perfect balance of ETH and TON, I started losing the gpu. My setup is just a gaming PC with a 1070 on 2nd slot that I mine on and also use for the display when mining (to improve LHR stability).
> 
> Frankly I don't remember if I had the gpu disconnection starting to occur right before I upgraded to 1.44 or after. But one thing is sure: The last 3 hours I've been trying to make v1.44 work but I keep encountering this issue after some time (or very quickly at high frequencies like 1600mhz).
> 
> I reverted now to 1.43 and it's been mining away very stable for 30mins+ at 1500mhz, giving 83.85Mh/s and 2.584Gh/s.
> 
> From the Nvidia forums I saw that gpus disappearing like that could be either bad thermals or the psu. As my card runs quite cool, core never exceeding 60c when mining and I had the pads changed for the mem, which makes it run at < 90c at high fan RPM. This leaves the power supply issue. But as now 1.43 is working fine, I'm thinking maybe there is some kind of spiky power usage happening on the newer version that makes the gpu fail hard and disconnect itself.
> 
> We need more people to shime in to be able to root out the cause of this. Also the youtuber CryptoDonkeyMiner had a similar thing happen, https://www.youtube.com/post/UgkxBsVSd7eK57eMFm3NTq2VaR9Sc5iJ7xiH He said he was testing ETH+ALPH where he noticed only one card showing, but after rebooting hiveOS he couldn't post, he had to check his rig and then he concluded that his 3080Ti is dead (similar to my issue, even bios couldn't recognize it anymore and the leds not turning on. I could only resurrect my card when I disconnect all power for a few minutes) CryptoDonkey had to test his card on another machine to finally see that it wasn't dead. This COULD be caused by lolminer if it was run before on that card or maybe the same bug on Trex.

lolMiner/Trex cannot die a GPU... that is a hardware problem, let's say not good Temperature, not right OC of the miner, riser problem mobo problem, eth...

**Comment by charnet3d** [comment]
Quick update: You were right, sorry to have jumped to conclusions faster (I was frustrated ^^'). I continued to have the issues on 1.43 as well just not as frequent. I switched the gpu with a 3080 and this one had the same issue again whereas the Ti worked fine in the other pc. After some more inspection I found the PCIe cable had a bit of burnt plastic around some pins in the psu side connector which could have made the connection worse or even cut off. I cleaned the pins and it seems to be working fine now (I also ordered new cables just in case)
